src/content/docs/git/2-multiplayer.mdx

```diff
@@ -0,0 +1,590 @@
+
+Repozytorium dodajemy komendą `git remote add <nazwa> <URL>`. Nazwa oczywiście musi być unikatowa.
+
+Na pobranie gałęzi jest kilka różnych sposobów.
+
+- Jeżeli nazwa gałęzi ze zmianami jest unikatowa (tj. nie występuje w żadnym innym repozytorium dodanym do twojego, w tym w twoim),
+  to wystarczy pobrać zdalne branche komendą `git fetch <nazwa repo>`, a następnie przełączyć się na brancha komendą `git checkout <nazwa brancha>`.
+  Git powinien sam utworzyć lokalnego brancha śledzącego brancha zdalnego.
+- Jeżeli nazwa nie jest unikatowa, powyższy sposób nie zadziałał, lub po prostu nie podoba ci się nazwa brancha zdalnego i chcesz użyć innej lokalnie,
+  to pobierz wybranego brancha bezpośrednio komendą `git fetch <nazwa repo> <nazwa zdalego brancha>:<nazwa docelowego lokalnego brancha>`.
+
+import review01 from "#assets/git/chapter-2/review-01.cast";
+
+<AsciinemaPlayer src={review01} idleTimeLimit={1} cols={100} rows={25} theme="monokai" >
+
+1. Otrzymujemy prośbę o sprawdzenie sugestii zmian
+2. trzeba odpowiedź wysłać
+3. Aktualizujemy informacje o zdalnym repozytorium
+4. Przełączamy się na nowego brancha `feat/epickie-ficzery`
+5. Sprawdzamy historię
+
+</AsciinemaPlayer>
+
+No dobra, mamy brancha... ale jakie zmiany zostały dokonane?
+
+Na początek warto sprawdzić listę dodanych zapisów.
+Komenda `git log` przyjmuje zakres zapisów, które ma wyświetlić.
+Na przykład, by sprawdzić listę zapisów dodanych od brancha `main`, można użyć komendy `git log main..`.
+
+Następnie przydałoby się sprawdzić w jakich plikach dokonano zmiany, ale bez szczegółów.
+Można to zrobić komendą `git diff --compact-summary <od>..[do]`.
+Na przykład, by zobaczyć jakie zmiany dokonano od brancha `main` do aktualnego zapisu należy wykonać komendę `git diff --compact-summary main..`.
+
+Gdy mamy już ogólny opis zmian, możemy sprawdzić jakie dokładnie zmiany nastąpiły w wybranych plikach.
+To też zrobimy komendą `git diff`, ale z lekko innymi parametrami - tym razem `git diff <od>..[do] [plik 1] [plik 2] ... [plik n]`.
+Na przykład, by przejrzeć zmiany w pliku `README.md` od brancha `main`, wykonamy komendę `git diff main.. README.md`.
+Jeżeli zmian jest mało i chcemy je wszystkie zobaczyć na raz, wystarczy pominąć listę plików.
+
+Można również przeglądać całe zapisy pojedynczo - komenda `git show <hash zapisu>` pokaże wszystkie dane o zapisie - w tym jego wiadomość i zmiany.
+
+import review02 from "#assets/git/chapter-2/review-02.cast";
+
+<AsciinemaPlayer src={review02} idleTimeLimit={1} cols={100} rows={25} theme="monokai" >
+
+1. Sprawdzamy liste nowych zapisów
+2. Generujemy krótkie podsumowanie zmian
+3. Sprawdzamy jakie zmiany nastąpiły w pliku `jakis_skrypt.sh`
+4. Sprawdzamy jakie zmiany nastąpiły w pliku `readme.md`
+5. Sprawdzamy szczegóły pierwszego zapisu
+6. Sprawdzamy szczegóły drugiego zapisu
+7. Sprawdzamy szczegóły ostatniego zapisu
+
+</AsciinemaPlayer>
+
+## Integracja
```

Comment:
brakuje mi na koniec jakiejś opinii czego powinno się używać w Solvro

Reply:
dodałem - w sumie to jest bardziej takie "używaj czego chcesz, ale oto kiedy której metody warto użyć" bo imo wybór metody integracji jest wyborem subiektywnym